cmd/beekeeper/cmd/check_fileretrieval.go

```diff
@@ -43,13 +44,16 @@ and attempts retrieval of those files from the last node in the cluster.`,
 				DebugAPIHostnamePattern: c.config.GetString(optionNameDebugAPIHostnamePattern),
 				DebugAPIDomain:          c.config.GetString(optionNameDebugAPIDomain),
 				DebugAPIInsecureTLS:     insecureTLSDebugAPI,
+				DisableNamespace:        disableNamespace,
 				Namespace:               c.config.GetString(optionNameNamespace),
 				Size:                    c.config.GetInt(optionNameNodeCount),
 			})
 			if err != nil {
 				return err
 			}
 
+			pusher := push.New(c.config.GetString(optionNamePushGateway), c.config.GetString(optionNameNamespace))
```

Comment:
Pushed does not have to constructed if metrics are not supposed to be pushed.

Reply:
as stated [here](https://github.com/ethersphere/beekeeper/pull/25#discussion_r444263520) `pusher` is used in too many places, after release we will refactor what is needed

Reply:
ok